data/events.json

```diff
@@ -0,0 +1,492 @@
+  },
+  {
+    "eventDbId": "6000000066",
+    "studyDbId": "6000000010",
+    "eventDescription": "sprinkler irrigation after planting to facilitate seedling emergence.",
+    "date": {
+      "discreteDates": ["2017-04-21", "2017-04-22", "2017-04-23", "2017-04-24", "2017-04-25", "2017-04-26", "2017-04-27", "2017-04-28", "2017-04-29", "2017-04-30", "2017-05-01", "2017-05-02", "2017-05-03", "2017-05-04"]
+    },
+    "eventParameters": [
+        {
+          "code": "IROP",
+          "name": "irrigation_operation",
+          "value": "IR004",
+          "unit": "code",
+          "codeValueDescription": "Sprinkler"
+        },
+        {
+          "code": "IRRIG",
+          "name": "irrigation_applied",
+          "value": "y"
+        },
+        {
+          "code": "IRVAL",
+          "name": "irrig_amount_depth",
+          "parameterDescription":"Irrigation amount, depth of water",
+          "unit": "mm",
+          "valuesByDate": ["4.5", "2.6", "0.4", "27.9", "14", "0", "14", "0", "14", "0", "14", "0", "14", "0", "2.7", "2.7", "14"]
+        }
```

Comment:
I'm not sure what happened here, you have 14 dates and 17 values

Reply:
wow, thanks for counting! Not sure how I got it wrong, but was missing the first three days. Original data in https://github.com/cct-datascience/ardn-terra-ref/blob/b0b8c1b3cc21233d1d65710273f86b54c34f53de/irrigation/season_4_irrigation.csv